### DISCUSSION TITLE
Importing from BigOven

### DISCUSSION THREAD

**Discussion by roertel** [TandoorRecipes/recipes#2875] [Q&A] [answered]
I have all my recipes in a very old version of the [BigOven](https://www.bigoven.com/) Windows application. It provides the ability to export recipes to HTML for dumping to a web page, presumably. These recipes don't import as is to Tandoor, but I found an HTML template file (see attached) in the directory tree that I *think* I can modify to affect the exported file. 

What is the best way for me to modify the HTML template in order to be able to import all my recipes into Tandoor? I was thinking to mimic the Paprika file, but it seems pretty different, so I thought I could ask around first. I can import the HTML directly or I can host the HTML on a localhost web server for importing, whatever works. 

HTML template file: [recipetheme0.txt](https://github.com/TandoorRecipes/recipes/files/13923896/recipetheme0.txt)
sample exported file: [Applebee_s_Oriental_Chicken_Salad_296.txt](https://github.com/TandoorRecipes/recipes/files/13923884/Applebee_s_Oriental_Chicken_Salad_296.txt)
(both files are .htm, but that's not a supported file)

**Comment by smilerz** [comment, marked as answer]
In order for an HTML file to be imported there must be a ld+json document in the page
https://developers.google.com/search/docs/appearance/structured-data/recipe

Otherwise you can attempt to copy any of the other structure imports.
